Modules/_testcapi/vectorcall_limited.c

```diff
@@ -1,3 +1,16 @@
+#include "pyconfig.h"  // Py_TRACE_REFS
+
+#ifdef Py_TRACE_REFS
+
+// Py_TRACE_REFS is incompatible with Limited API
+#include "parts.h"
+int
+_PyTestCapi_Init_VectorcallLimited(PyObject *m) {
```

Comment:
I suggest putting an #ifdef on the _PyTestCapi_Init_VectorcallLimited() call in _testcapimodole.c instead of defining a no-op function.

Maybe define somewhere a macro to decide if vectorcall is tested or not? Something like:

```
#ifndef Py_TRACE_REFS
#  define TEST_VECTORCALL
#endif
```